cypress/integration/vitessce-data-uri.spec.js

```diff
@@ -0,0 +1,89 @@
+
+import { urlPrefix } from '../../src/app/api';
```

Comment:
Is this used somewhere?

Reply:
no: Just removed, along with the setup code below. Thanks!